File: uri/calculosimples/calculosimples.kt
import java.util.*
import kotlin.text.*

fun main( args: Array<String> ){
    var (codeA, npecasA, valorA) = readLine()!!.split(' ') 
    var (codeB, npecasB, valorB) = readLine()!!.split(' ')
    var foo = (  npecasA.toInt() * valorA.toDouble() ) + (npecasB.toInt()  * valorB.toDouble() )
    println("VALOR A PAGAR: R$ %.2f".format(foo))
}

File: uri/calculosimples/calculosimples.lua
codeA, npecasA, valorA = io.read("*n","*n","*n")
codeB, npecasB, valorB = io.read("*n","*n","*n")
foo = string.format('%.2f', (npecasA * valorA) + (npecasB * valorB ))
io.write('VALOR A PAGAR: R$ ', foo ,'\n' )